Authorization/SignIn/UserClaimsGatherer.cs

```diff
@@ -28,7 +29,10 @@ public IEnumerable<Claim> Gather(IUser user)
                 AddClaimsFromGroup(userGroup, dbClaims);
             }
 
-            return dbClaims.Select(_claimDbConverter.Unpack);
+            var claims = dbClaims.Select(_claimDbConverter.Unpack).ToList();
```

Comment:
Hi @miyconst, maybe you have a better idea how to implement this feature?
I mean adding claim with type `ClaimTypes.Name` containing `Username` of the signed in user as a value for the possibility to use it for authorization purposes in the apps?